### DISCUSSION TITLE
Reading Bold/Italicized Text

### DISCUSSION THREAD

**Discussion by UnfeaturedPROD** [GrandyB/SheetsIO#44] [Ideas]
Hey!

As the title suggests, my idea is for SheetsIO to display bold/italicized text. Is this possible? It could make many of the elements I incorporate into my esports productions pop out more. As I'm writing this I'm slowly starting to think as to how this couldn't work, since I'm pretty sure regular .txt files don't have a way of simply bolding or italicizing text (maybe unless you change the font family of the .txt file).

I've worked with a few tools to help with my side gig (producing esports twitch streams) and SheetsIO is by far the best free software I have seen. It's incredibly simple to set up and use which is a big plus when getting fresh talent. This is just the only thing I see other production tools having over SheetsIO, but I think the reason why this works in the tool shown below is because it launches a server from your PC which reads images and text files and then sets them up accordingly depending on the text written in the file.

https://cdn.discordapp.com/attachments/318122927902162945/1049194750831509574/image.png

Thanks anyways!

**Comment by GrandyB** [comment]
Hello there!
There wouldn't be a way for SheetsIO to specify it sadly.

The BGG tool and some others basically have HTML rendering engines they use to display stuff; SheetsIO is less complex and can only grab text from a sheet and slap it into a file. You set what font, whether it's italic etc when adding the text source on the OBS side instead.
